ONSdigital/response-operations-ui
# collection exercise event status display

# What and why?

The purpose of this PR is to display collection exercise event status to a user via the interface on the survey list and collection exercise details page.

# How to test?

* Create a CE and fill out event dats for MPS, go live, reminders and nudge emails.
* In the database, edit some of the event statuses from `SCHEDULED` to `RETRY`, `PROCESSING`, and/or `FAILED`.
* Make sure event status indicators are appearing as intended (look at scenarios on Trello card).

NOTE: Because of a bug unrelated to this card, you can't currently create a fifth nudge email for a CE without causing a 500 error, but you can create the first four, and HTML code has been supplied to handle a fifth nudge email, so that one should be okay assuming the other events are okay.

# Trello
[Card](https://trello.com/c/qV7f3B9M)
